src/prefect/orion/schemas/schedules.py

```diff
@@ -398,7 +398,7 @@ def validate_rrule_str(cls, v):
             raise ValueError(f'Invalid RRule string "{v}": {exc}')
         if len(v) > MAX_RRULE_LENGTH:
             raise ValueError(
-                f'Invalid RRule string "{v}"\n'
+                f'Invalid RRule string "{v[:40] + "..." if len(v) > 40 else v}"\n'
```

Comment:
Don't we know the length is > 40 since we're in this clause?

Reply:
I think I was thinking that `MAX_RRULE_LENGTH` could technically be anything? You're correct though, just simplified it

Reply:
True but practically speaking 🤷‍♂️ not sure what's best practice in this case.